libnavigation-ui/src/main/res/layout/layout_speed_limit.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    android:id="@+id/speedView"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    xmlns:app="http://schemas.android.com/apk/res-auto">
+
+    <ImageView
+        android:id="@+id/speedLimit"
+        android:layout_width="0dp"
+        android:layout_height="0dp"
+        app:layout_constraintTop_toTopOf="parent"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintBottom_toBottomOf="parent" />
+
+    <TextView
+        android:id="@+id/speedLimitText"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_margin="20dp"
+        app:layout_constraintTop_toTopOf="@id/speedLimit"
+        app:layout_constraintEnd_toEndOf="@id/speedLimit"
+        app:layout_constraintStart_toStartOf="@id/speedLimit"
+        app:layout_constraintBottom_toBottomOf="@id/speedLimit"
+        android:textSize="16sp"
+        android:textColor="#333333"
```

Comment:
Are we hardcoding the `textColor`? Should we make that configurable? In any case, we should probably extract that out into `colors.xml`.

Reply:
Done.